server.py

```diff
@@ -180,10 +180,10 @@ def get_place_categories_json():
 
     categories = crud.get_place_categories()
 
-    list_categories = []
+    list_categories = [category.place_category_id for category in categories]
```

Comment:
The way you did it is totally fine. Just wanted to point out an example of list comprehension